### PR TITLE
remove accept, fix spelling

### PR DESCRIPTION
**Pull request by gruiz17**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -47,8 +47,7 @@ app.use(require('cookie-session')({
 app.use(passport.initialize());
 app.use(passport.session());
 
-// custom mrepodleware
+// custom middleware
 app.use('/api', require('./middleware/param'));
 app.use('/github', require('./middleware/param'));
-app.use('/accept', require('./middleware/param'));
```
Comment (dfarr) [inline app.js]: This is a required middleware, do not remove !fix

File: app.js [hunk 2]
```diff
@@ -47,8 +47,7 @@ app.use(require('cookie-session')({
 app.use(passport.initialize());
 app.use(passport.session());
 
-// custom mrepodleware
+// custom middleware
```
Comment (dfarr) [inline app.js]: middleware > mrepodleware !fixed

File: app.js (hunk 1 above)
Comment (gruiz17) [inline app.js]: i re-added accept in the latest commit! !fixed

File: app.js (hunk 2 above)
Comment (dfarr) [inline app.js]: this is not! !resolve

**Comment by dfarr** [comment]
:+1:

**Comment by dfarr** [comment]
-1

File: app.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ app.use(require('cookie-session')({
...
```
Comment (gruiz17) [inline app.js]: !fixed

Comment (gruiz17) [inline app.js]: !resolve

File: app.js (hunk 1 above)
Comment (gruiz17) [inline app.js]: !resolve

File: app.js (hunk 3 above)
Comment (gruiz17) [inline app.js]: !fixed

File: app.js (hunk 1 above)
Comment (gruiz17) [inline app.js]: !fixed

**Pull request merged**